cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/buildah/buildah-task.yaml

```diff
@@ -71,10 +78,21 @@ spec:
     image: $(params.BUILDER_IMAGE)
     workingDir: $(workspaces.source.path)
     script: |
-      buildah --storage-driver=$(params.STORAGE_DRIVER) bud \
-        $(params.BUILD_EXTRA_ARGS) --format=$(params.FORMAT) \
-        --tls-verify=$(params.TLSVERIFY) --no-cache \
-        -f $(params.DOCKERFILE) -t $(params.IMAGE) $(params.CONTEXT)
+      if [[ "$(workspaces.rhel-entitlement.bound)" == "true" ]]; then
+        mkdir /tmp/entitlement
+        cp $(workspaces.rhel-entitlement.path)/entitlement.pem /tmp/entitlement
```

Comment:
do we need to copy? can't we mount from the workspace location itself?

Reply:
if we mount directly i am facing issue related to permission

I mean i tried without copy 
```
buildah bud --volume $(workspaces.rhel-entitlement.path):/etc/pki/entitlement $COMMON_BUILD_ARGS
```
the build step failed with permission error

```
[buildah : build-and-push] STEP 2/8: RUN dnf search kernel-devel --showduplicates | tail -n2
[buildah : build-and-push] time="2024-01-31T12:20:36Z" level=warning msg="Path \"/run/secrets/etc-pki-entitlement\" from \"/etc/containers/mounts.conf\" doesn't exist, skipping"
[buildah : build-and-push] time="2024-01-31T12:20:36Z" level=warning msg="pkg/bind: error detaching \"/var/tmp/buildah312009441/mnt/buildah-bind-target-10\": permission denied"
[buildah : build-and-push] time="2024-01-31T12:20:36Z" level=warning msg="pkg/bind: error removing \"/var/tmp/buildah312009441/mnt/buildah-bind-target-10\": device or resource busy"
[buildah : build-and-push] time="2024-01-31T12:20:36Z" level=warning msg="pkg/bind: error removing \"/var/tmp/buildah312009441/mnt\": directory not empty"
[buildah : build-and-push] error running subprocess: remounting "/var/tmp/buildah312009441/mnt/rootfs/etc/pki/entitlement" in mount namespace with flags 0x0 instead of 0x1: permission denied
[buildah : build-and-push] Error: building at STEP "RUN dnf search kernel-devel --showduplicates | tail -n2": exit status 1
```

Reply:
can we use the mountPath field in workspace to mount at specific location

Reply:
lets merge it now and try to fix warning in a followup PR